src/libs/antares/study/parts/hydro/container.cpp

```diff
@@ -132,235 +132,61 @@ bool PartHydro::LoadFromFolder(Study& study, const AnyString& folder)
-            area.hydro.reservoirLevel.reset(3, DAYS_PER_YEAR, true);
-            for (int x = area.hydro.minimum; x <= area.hydro.maximum; x++)
-            {
-                auto& col = area.hydro.reservoirLevel[x];
-                for (int i = 0; i < DAYS_PER_YEAR; i++)
-                {
-                    col[i] = temp[x][i];
-                }
-            }
+            ret = area.hydro.maxPower.loadFromCSVFile(
+                    buffer, 4, DAYS_PER_YEAR, Matrix<>::optFixedSize, &study.dataBuffer)
+                && ret;
 
             if (enabledModeIsChanged)
                 JIT::enabled = true; // Back to the previous loading mode.
         }
         else
         {
-            area.hydro.reservoirLevel.reset(3, DAYS_PER_YEAR, true);
-            area.hydro.reservoirLevel.fillColumn(area.hydro.minimum, 0.0);
-            area.hydro.reservoirLevel.fillColumn(area.hydro.average, 0.5);
-            area.hydro.reservoirLevel.fillColumn(area.hydro.maximum, 1.);
-            area.hydro.reservoirLevel.markAsModified();
+            ret = area.hydro.maxPower.loadFromCSVFile(
+                    buffer, 4, DAYS_PER_YEAR, Matrix<>::optFixedSize, &study.dataBuffer)
+                && ret;
         }
 
-        if (study.header.version >= 620)
-        {
-            buffer.clear() << folder << SEP << "common" << SEP << "capacity" << SEP
-                           << "waterValues_" << area.id << '.' << study.inputExtension;
-            ret = area.hydro.waterValues.loadFromCSVFile(
-                    buffer, 101, DAYS_PER_YEAR, Matrix<>::optFixedSize, &study.dataBuffer)
-                  && ret;
-        }
-        else
-        {
-            area.hydro.waterValues.reset(101, DAYS_PER_YEAR, true);
-            area.hydro.waterValues.markAsModified();
-        }
+        buffer.clear() << folder << SEP << "common" << SEP << "capacity" << SEP
+                       << "creditmodulations_" << area.id << '.' << study.inputExtension;
+        ret = area.hydro.creditModulation.loadFromCSVFile(
+                buffer, 101, 2, Matrix<>::optFixedSize, &study.dataBuffer)
+              && ret;
 
-        if (study.header.version >= 620)
-        {
-            buffer.clear() << folder << SEP << "common" << SEP << "capacity" << SEP
-                           << "inflowPattern_" << area.id << '.' << study.inputExtension;
-            ret = area.hydro.inflowPattern.loadFromCSVFile(
-                    buffer, 1, DAYS_PER_YEAR, Matrix<>::optFixedSize, &study.dataBuffer)
-                  && ret;
-        }
-        else
-        {
-            area.hydro.inflowPattern.reset(1, DAYS_PER_YEAR, true);
-            area.hydro.inflowPattern.fillColumn(0, 1.0);
-            area.hydro.inflowPattern.markAsModified();
-        }
+        buffer.clear() << folder << SEP << "common" << SEP << "capacity" << SEP << "reservoir_"
+                       << area.id << '.' << study.inputExtension;
+        ret = area.hydro.reservoirLevel.loadFromCSVFile(
```

Comment:
For reservoir levels loading, block : 
```C++
if (JIT::enabled)
{
    ...
}
...
if (enabledModeIsChanged)
     JIT::enabled = true;
```
disappeared. 
Is it normal ?

Reply:
```cpp
if (study.header.version >= 620)
{
    // ...
}
else if (study.header.version >= 390)
{
    bool enabledModeIsChanged = false;
    if (JIT::enabled)
    {
        JIT::enabled = false; // Allowing to read the area's daily max power
        enabledModeIsChanged = true;
    }
    buffer.clear() << folder << SEP << "common" << SEP << "capacity" << SEP << "reservoir_"
                   << area.id << '.' << study.inputExtension;
    ret = area.hydro.reservoirLevel.loadFromCSVFile(
            buffer, 3, 12, Matrix<>::optFixedSize, &study.dataBuffer)
          && ret;
    // ...
}
```
I think that this code block was only executed in versions 390-610, so it was removed with the rest.